src/unix/thread.c

```diff
@@ -171,6 +171,11 @@ int uv_thread_create_ex(uv_thread_t* tid,
   f.in = entry;
   err = pthread_create(tid, attr, f.out, arg);
 
+  if (params->flags & UV_THREAD_DETACHED) {
```

Comment:
Since we are already using an attribute, we could use this:

pthread_attr_setdetachstate(&attr, PTHREAD_CREATE_DETACHED);

Reply:
Also note that this code path has a bug: it calls pthread_detach unconditionally, regardless of whether pthread_create succeeded or failed.

The Windows code has the same bug.

Reply:
Will address this